stmt/expression.go

```diff
@@ -16,20 +15,13 @@ type Expression interface {
 
 // NewExpression returns a new Expression instance from arg.
 func NewExpression(arg interface{}) Expression { // nolint: gocyclo
-	if arg == nil {
-		return NewValue(nil)
-	}
-
 	switch value := arg.(type) {
 	case Expression:
 		return value
-	case string, bool, int, int8, int16, int32, int64,
```

Comment:
Because I don't want some weird behavior with `XEncoder` or any other struct, can you keep the type assertions on native type and keep the panic on unexpected types ?

Thank you.